appvia/kore
# reference the correct credentials type for route53 domains management

## Summary

When provisioning an automated route53 child domain when using automated AWS account provisioning, the domain would only get as far as an "Error" status.

This PR fixes this issue.

**Which issue(s) this PR resolves**:
Resolves #1365 

## Changes

We simply reference the eks provider specific credential and not the AccountManagement "cluster" level credential.

## Examples

NA

## Testing

In an environment where AWS account automation is used:

1. Setup a organisation level domain and complete delegation
2. Create an EKS cluster
3. Under the DNS tab for a team, ensure the child domain status goes to "Success"
